public/assets/scss/_search.scss

```diff
@@ -131,50 +132,63 @@
         }
     }
     &-initial {
-        @mixin light() {
-            background: $yellow-mid;
-        }
-        @mixin base() {
+        @mixin track() {
             background: $yellow;
         }
+        @mixin thumb {
```

Comment:
Can we call this yellow thumb so that it can be more descriptive and easily differentiated from the green thumb below?